src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -125,18 +128,34 @@ public void LogState()
             builder.AppendLine("User provided? - " + (User != null));
             var dict = MsalHelpers.ParseKeyValueList(ExtraQueryParameters, '&', true, RequestContext);
             builder.AppendLine("Extra Query Params Keys (space separated) - " + dict.Keys.AsSingleString());
+#if DESKTOP || NETSTANDARD1_3
+            builder.AppendLine("Confidential Client? - " + (ClientCredential != null));
+            builder.AppendLine("Client Credential Request? - " + IsClientCredentialRequest);
+            if (IsClientCredentialRequest)
+            {
+                builder.AppendLine("Client Certificate Provided? - " + (ClientCredential.Certificate != null));
+            }
+#endif
+            RequestContext.Logger.InfoPii(builder.ToString());
+
+            // Create no Pii enabled string builder
+            builder = new StringBuilder(Environment.NewLine + "=== Request Data ===" + Environment.NewLine +
+                                        "Authority Provided? - " + (Authority != null) + Environment.NewLine);
+            builder.AppendLine("Scopes - " + Scope?.AsSingleString());
+            builder.AppendLine("Validate Authority? - " + ValidateAuthority);
+            builder.AppendLine("LoginHint provided? - " + !string.IsNullOrEmpty(LoginHint));
+            builder.AppendLine("User provided? - " + (User != null));
             dict = MsalHelpers.ParseKeyValueList(ExtraQueryParameters, '&', true, RequestContext);
-            builder.AppendLine("Slice Parameters Keys(space separated) - " + dict.Keys.AsSingleString());
+            builder.AppendLine("Extra Query Params Keys (space separated) - " + dict.Keys.AsSingleString());
```

Comment:
why are you adding same log in again?

Reply:
fixed.

Reply:
I added in the slice parameters, as per our conversation